fiftyone/utils/clip/zoo.py

```diff
@@ -101,13 +128,10 @@ def _load_network(self, config):
 
         return build_model(model.state_dict()).to(self.device).float()
 
-    def _prepare_text_features(self):
+    def _embed_prompts(self, prompts):
```

Comment:
`_text_features` make more sense than `embed_prompt`. Can we just use the current name?